auslib/web/balrog_api.py

```diff
@@ -0,0 +1,76 @@
+import connexion
+from connexion.operation import Operation
+from connexion.resolver import Resolver
+
+
+RANDOM_DIGITS = 6
+
+
+class BalrogApi(connexion.Api):
```

Comment:
Ah, I see! Is there a way to do it without replicating all this code from Connexion? Unfortunately, this code is a maintenance nightmare, it will require updates whenever Connexion internals change :(. It's actually better to use separate methods for endpoint than do this.